src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyShares.tsx

```diff
@@ -3,38 +3,35 @@ import Grid from '@mui/material/Grid';
 import { observer } from 'mobx-react';
 import Typography from '@mui/material/Typography';
 import { KeyShares, SSVKeysException } from 'ssv-keys';
+import { useLocation, useNavigate } from 'react-router-dom';
```

Comment:
Generally this file looks quite long, we should consider if it can be broken down to different separate components.

Reply:
+